aggregator_core/src/datastore/tests.rs

```diff
@@ -107,18 +107,8 @@ async fn roundtrip_task(ephemeral_datastore: EphemeralDatastore) {
         (VdafInstance::Prio3CountVec { length: 64 }, Role::Helper),
         (VdafInstance::Prio3Sum { bits: 64 }, Role::Helper),
         (VdafInstance::Prio3Sum { bits: 32 }, Role::Helper),
-        (
-            VdafInstance::Prio3Histogram {
-                buckets: Vec::from([0, 100, 200, 400]),
-            },
-            Role::Leader,
-        ),
-        (
-            VdafInstance::Prio3Histogram {
-                buckets: Vec::from([0, 25, 50, 75, 100]),
-            },
-            Role::Leader,
-        ),
+        (VdafInstance::Prio3Histogram { buckets: 4 }, Role::Leader),
+        (VdafInstance::Prio3Histogram { buckets: 5 }, Role::Leader),
```

Comment:
Technically the direct translation of these parameters would be 5 and 6 buckets, respectively, but it doesn't really matter for the purposes of checking serialization/deserialization.